bin/MadGraph5_aMCatNLO/cards/production/13TeV/ST_FCNC/TA_Tleptonic_kappa_act_LO/TA_Tleptonic_kappa_act_LO_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+# Model
+import model TopFCNC_UFO-kappa_act_ProdConventions_no_c_mass --modelname
+ 
+# Multiparticles
+define l+  = e+ mu+ ta+
+define l-  = e- mu- ta-
+
+generate    p p > t  a, (t  > w+ b,  w+ > l+ vl ) NP=1 @10
+add process p p > t~ a, (t~ > w- b~, w- > l- vl~) NP=1 @20
```

Comment:
The new physics term will come from the vertex "p p > t a" right?
Then I think "p p > t a NP=1, (t  > w+ b,  w+ > l+ vl )" should be correct? @agrohsje

Reply:
Moving NP=1 to t a makes it explicit. With the given model there is no impact.